cogs/random_resource.py

```diff
@@ -0,0 +1,47 @@
+import discord
+from discord import app_commands
+from discord.ext import commands
+import random
+
+from api import get_random_resource
+
+
+class RandomResourceCog(commands.Cog):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @app_commands.command(
+        name="random-resource", description="Send a random WIT resource!"
+    )
+    async def send_random_resource(self, int: discord.Interaction):
+        link_format = "https://unswwit.com/media/"
```

Comment:
Could you make this into a constant with all caps, similar to this
`EVENT_RECAPS_LINK = "https://unswwit.com/events/event-recaps/"`, move this to after line 6 and perhaps rename it to `MEDIA_LINK`